security/network_bound_disk_encryption/nbde-installing-a-tang-server.adoc

```diff
@@ -0,0 +1,23 @@
+// CNF-2127 assembly
+[id="nbde-installing-a-tang-server"]
+= Tang server installation considerations
```

Comment:
I think this file is a duplicate of nbde-tang-server-installation-considerations.adoc. Pick the one that you prefer.

Reply:
Removing nbde-installing-a-tang-server.adoc